src/restLink.ts

```diff
@@ -441,13 +460,23 @@ const resolver: Resolver = async (
     }
 
     validateRequestMethodForOperationType(method, operationType || 'query');
-
     return await (customFetch || fetch)(`${uri}${pathWithParams}`, {
       credentials,
       method,
       headers,
       body: body && JSON.stringify(body),
     })
+      .then(res => {
+        if (res.status >= 300) {
```

Comment:
This raises an interesting question: Should we follow 301/302 redirects automatically, internally?

Reply:
I think this raises a "Feature Request" that we need to have a link-level fetchOptions so people can configure the number of redirects to follow.

Reply:
Good idea @fbartho! Axios uses a property **maxRedirects** to handle this. We could have something like this:
`fetchOptions: {
maxRedirects: 5
}`

Can/should we handle this in a separate PR?

Reply:
@jslowack yes, I support that, yes I agree that makes sense as a separate PR. That said, I recommend we exactly implement the official spec for fetchOptions, that way we don’t have a translation layer.

Reply:
Oké, I'll create a PR (using https://fetch.spec.whatwg.org/#concept-request-redirect-mode)